lib/kubernetes-deploy/runner.rb

```diff
@@ -290,30 +290,23 @@ def confirm_context_exists
     end
 
     def confirm_namespace_exists
-      _, _, st = run_kubectl("get", "namespace", @namespace, namespaced: false)
+      _, _, st = run_kubectl("get", "namespace", @namespace, with_namespace: false)
```

Comment:
Why rename this to `with_namespace`? `namespaced` seemed like a better name, `with_*` usually implies a block to me.